dashboard/app/models/lesson.rb

```diff
@@ -765,7 +764,11 @@ def copy_to_script(destination_script)
           copied_activity_section.key = SecureRandom.uuid
           copied_activity_section.lesson_activity_id = copied_lesson_activity.id
           copied_activity_section.save!
-          sl_data = original_activity_section.script_levels.map.with_index(1) {|l, pos| JSON.parse({assessment: l.assessment, bonus: l.bonus, challenge: l.challenge, levels: l.levels, activitySectionPosition: pos}.to_json)}
+          sl_data = original_activity_section.script_levels.map.with_index(1) do |original_script_level, pos|
+            original_active_level = original_script_level.oldest_active_level
+            copied_level = new_level_suffix.blank? ? original_active_level : original_active_level.clone_with_suffix(new_level_suffix)
+            JSON.parse({assessment: original_script_level.assessment, bonus: original_script_level.bonus, challenge: original_script_level.challenge, levels: [copied_level], activitySectionPosition: pos}.to_json)
```

Comment:
do we need to also copy progression on this line too?

Reply:
oh yes, probably. I think I copied this logic from the import script which was from when we were getting the progression from the activity section

Reply:
oh nevermind, the [progression name](https://github.com/code-dot-org/code-dot-org/blob/e394e6c50bcb358e5ab070e34ff79c3dd03ea36f/dashboard/app/models/activity_section.rb#L42) is set on the activity section which should get copied over with `dup`